experiment-setup/execution/host_sequence/scenario_api.py

```diff
@@ -28,12 +27,13 @@ def request_endpoint(path, method="get", data=None, params=None, user_session=No
 
 def sequential_interval_scenario(service, start, iter):
     # set main user
```

Comment:
I would like to change the collection of the data here, else wise we would need to normalize the data in a annoying way. Initialize an empty dataframe here `results=[]'

In the `request_endpoint` func lets change the return path to `endpoint: f"/api{path}"` so we know if the endpoint is from the api or the page since we change the datastructures.